<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBoxGadget</class>
 <widget class="QGroupBox" name="GroupBoxGadget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QGroupBox{
  background-color: transparent;
  background:#0a0a0a;
  border: 2px solid white; 
  border-radius: 5px;
}</string>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonClockwise">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/rotate_anti_clockwise.png</normaloff>:/icons/transform/rotate_anti_clockwise.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonUp">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/translate_up.png</normaloff>:/icons/transform/translate_up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonAntiClockwise">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/rotate_clockwise.png</normaloff>:/icons/transform/rotate_clockwise.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonLeft">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/translate_left.png</normaloff>:/icons/transform/translate_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonDown">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/translate_down.png</normaloff>:/icons/transform/translate_down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonRight">
     <property name="styleSheet">
      <string notr="true">QPushButton::pressed { 
	padding-left:6px;
	padding-top:6px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/transform/translate_right.png</normaloff>:/icons/transform/translate_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
